core/nodejsAction/Dockerfile

```diff
@@ -52,6 +52,7 @@ swagger-tools@0.8.7 \
 tmp@0.0.28 \
 watson-developer-cloud@1.4.1 \
 when@3.7.3 \
+ws@1.1.0 \
```

Comment:
@bjustin-ibm go ahead and add the new npm package in both places

1.  [docs/reference.md#runtime-environment](https://github.com/openwhisk/openwhisk/blob/master/docs/reference.md#runtime-environment) 
2.  [openwhisk/openwhisk_reference.md#runtime-environment](https://github.com/IBM-Bluemix/docs/blob/master/openwhisk/openwhisk_reference.md#runtime-environment)